### ISSUE TITLE
Run with `@debug`  and `@announce` tag fails with `#to_h` missing

### ISSUE THREAD

**Issue by maxmeyer** [cucumber/aruba#563]
## Summary

Running a feature with `@debug`  and `@announce` tag fails with:

~~~
<<-COMMAND
NotImplementedError
COMMAND

undefined method `to_h' for NotImplementedError:Class
Did you mean?  to_s (NoMethodError)
~~~

## Expected Behavior

Command execution should not fail with ruby error.

## Current Behavior

The execution fails with the following error.

~~~
undefined method `to_h' for NotImplementedError:Class
Did you mean?  to_s (NoMethodError)
~~~

## Possible Solution

If I remember correctly, I forgot to add `NotImplementedError#to_h`. Maybe returning just an empty hash is a solution for this. I'm not sure, where we use this class.

## Steps to Reproduce (for bugs)

1. Create feature file
~~~
Feature: My feature
  @debug @announce
  Scenario: Test it
    Given I run `hello world` interactively
    Then the stderr should contain:
    """
    hello world
    """
~~~
2. Create `step_definitions.rb`
~~~
require 'aruba/cucumber'
~~~

3. Run cucumber

## Your Environment

* Version used: 1.0.0.pre.alpha.2
* Operating System and version: Ruby
* Link to your project: internal

**Comment by luke-hill** [comment]
I'll investigate this issue @mvz as it seems easy enough to repro.

Quick question, is this just to be cut against master or against master and still?

**Comment by mvz** [comment]
Looks like it's a problem that only occurs on master, but we'll see.

**Comment by luke-hill** [comment]
I've got it about 90% fixed on master. But interestingly the debug flag masks things in a different way. But I've found where the bug is exposed. Just working out the best way to fix it.

**Comment by mvz** [comment]
Fixed in #664 and #665.
